layouts/html.hbs

```diff
@@ -116,11 +116,10 @@
       {{#babel}}
       function iframeGetSearchParams() {
         let params = new URLSearchParams(window.location.search);
-        let verticalUrl = window.location.pathname.substr(1)
-        if (verticalUrl) {
-          params.set('verticalUrl', verticalUrl);
-        }
-        return params.toString();
+        let verticalUrl = window.location.pathname.substr(1);
```

Comment:
If we're not using `params.set`, do we need to wrap the `window.location.search` in `URLSearchParams` any more? Could we instead just have:
```
let params = window.location.search;
return verticalUrl  ? params + '&verticalUrl=' ...
```

Reply:
ah yup, that's not needed any more. Updated